airflow/jobs/scheduler_job.py

```diff
@@ -1555,17 +1555,14 @@ def _process_executor_events(self, simple_dag_bag: SimpleDagBag, session: Sessio
             # TODO: should we fail RUNNING as well, as we do in Backfills?
             if ti.try_number == buffer_key.try_number and ti.state == State.QUEUED:
                 Stats.incr('scheduler.tasks.killed_externally')
-                self.log.error(
-                    "Executor reports task instance %s finished (%s) although the task says its %s. "
-                    "(Info: %s) Was the task killed externally?",
-                    ti, state, ti.state, info
-                )
+                msg = f"Executor reports task instance {ti} finished ({state}) although the " \
```

Comment:
Using f-string in logs to pasd parameters is generally not a good practice since Log formatters can change the parameters passed to a LogRecord

Reply:
Oh, that's true. I will use template string.